action/action_test.go

```diff
@@ -88,8 +88,14 @@ func TestAction(t *testing.T) {
 		t.Fatalf("Error: %s", err)
 	}
 
-	if as := act.String(); as != "Store(Path: "+filepath.Join(td, "store")+", Mounts: )" {
```

Comment:
this was failing on mac because of the '/private/' being on path for some reason, which is not there on mac.
so instead of checking `act.String()` i just checked what it prints out and did some explicit checks of those stuff.